test/rubocop/cop/minitest/skip_without_reason_test.rb

```diff
@@ -0,0 +1,114 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class SkipWithoutReasonTest < Minitest::Test
+  def test_registers_offense_when_skip_without_reason
+    assert_offense(<<~RUBY)
+      skip
+      ^^^^ Add a reason explaining why the test is skipped.
+    RUBY
+  end
+
+  def test_registers_offense_when_skip_with_empty_reason
+    assert_offense(<<~RUBY)
+      skip('')
+      ^^^^^^^^ Add a reason explaining why the test is skipped.
+    RUBY
+  end
+
+  def test_does_not_register_offense_when_skip_with_string_reason
+    assert_no_offenses(<<~RUBY)
+      skip('Reason')
+    RUBY
+  end
+
+  def test_does_not_register_offense_when_skip_with_dynamic_reason
+    assert_no_offenses(<<~RUBY)
+      skip(reason)
+    RUBY
+  end
```

Comment:
Can you add a test and update the implement for `obj.skip` that should be allowed?